### ISSUE TITLE
no admin access to some software

### ISSUE THREAD

**Issue by xarthurx** [lukesampson/scoop-extras#2604]
I'm trying to install teamviewer from scoop.

Tried install locally and run w/o admin access, or install globally and run w/o admin access.

However, some of the settings that need to be activated with admin access are still not available, for instance:

"keep logged in", or "Grand Easy Access".

Is there any difference about admin access of software installed from scoop?
Very confused.

**Comment by issaclin32** [comment]
I assume that some of TeamViewer's function requires **installing a driver or a Windows service**, which can only be done by installer. (the manifest here uses portable ZIP version)

[teamviewer-np.txt](https://github.com/lukesampson/scoop-extras/files/3488913/teamviewer-np.txt)

I wrote a manifest using the installer, see if this works for you.
Rename it to `teamviewer-np.json` and install it with `scoop install teamviewer-np.json`

If this works fine, I will send a pull request to [the nonportable bucket](https://github.com/oltolm/scoop-nonportable)

**Comment by xarthurx** [comment]
Yes, using the script works fine and issues mentioned above disappear.
Great, thanks!

**Comment by issaclin32** [comment]
Non-portable version of **TeamViewer** has been added to the nonportable bucket. You can install it by running `scoop bucket add nonportable` and `scoop install teamviewer-np`.
